plc4j/integrations/apache-nifi/nifi-plc4x-processors/src/main/java/org/apache/plc4x/nifi/util/Plc4xCommon.java

```diff
@@ -145,5 +155,100 @@ public static Schema createSchema(Map<String, ? extends PlcValue> responseDataSt
 
 	}
 	
-
+	
```

Comment:
@chrisdutz this seems very clumsy.  For almost all of these you just want to return whatever the "real" type of the value is.  Is there no function where you can get that?

Reply:
Still looking for @chrisdutz feedback